skymp5-client/src/skympClient.ts

```diff
@@ -57,7 +59,7 @@ const handleMessage = (msgAny: AnyMessage, handler_: MsgHandler) => {
   if (msgType === "hostStart") {
     const msg = msgAny as HostStartMessage;
     const target = msg.target;
-    printConsole("hostStart", target.toString(16));
+    //printConsole("hostStart", target.toString(16));
```

Comment:
`hostStart` is a really bad log message. I'm glad you removed it but please also remove the commented version of this log message.